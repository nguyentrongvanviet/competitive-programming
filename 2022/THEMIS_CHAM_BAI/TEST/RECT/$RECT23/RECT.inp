135
669226787 607305120
74124520 -844470490
690870776 -757961809
-938644522 -1289302
-206310355 -178548890
-301701574 861822987
824266467 607741844
219279380 789037896
-594378658 -546089373
-621809940 258140701
-493090869 191572913
604615124 -945513863
-476801362 621882393
955064144 518925679
133148193 917876621
696751429 -39190636
451831353 666320402
714835107 850405166
-493090869 191572913
-951900382 -114212461
696678259 -635313905
245116041 356484886
-787355559 -407819218
-141897045 -662091388
-910409553 749536821
-681335676 -103901456
826974452 -211517981
626675448 266289701
-549418582 704643703
-364284568 848438515
-620533574 -963907927
-791537475 -820899985
-37477677 -773262056
620017390 510171980
-392079950 -197441311
-252093471 977279162
-188781912 311548231
366274944 -984668399
98805823 542782645
-980375075 102184209
-370195240 153942798
931952899 480295495
-558410028 -718186525
309633859 -763927543
830948145 69184802
-134832003 -380449678
630236821 -456599127
683489441 983059756
98577910 -509798588
779748167 -751346431
-558410028 -718186525
-228132893 -852633985
-726898788 723113914
-915144805 -336640134
329189606 -561343510
325854839 502509396
98577910 -509798588
-203162910 -583501884
661946443 234405554
93253994 -773355018
145841160 600184422
-493890581 333465466
-299557575 977143839
-955307220 -190393226
924824877 -642279331
-140909764 -862666057
-549418582 704643703
683489441 983059756
-592088211 828734307
112391086 3860025
279673505 -299378958
-256813040 -585045887
-773888457 -936517148
646775445 497479852
219279380 789037896
-621809940 258140701
-143265268 -183430736
-757437541 490956008
905825654 -25940139
-784058612 743937824
120646279 -130934797
-45045016 271531452
-206310355 -178548890
-252093471 977279162
-866702977 418363423
74124520 -844470490
-710483166 552984333
514365680 -35008202
-702530100 781872282
329189606 -561343510
512017432 -606989172
970939134 -428045063
81616791 -138876528
-232550855 -533238589
840991529 -879207333
-109656365 -309515942
-955307220 -190393226
930588836 20794645
929442372 234845634
378033695 -328729662
970375323 175275526
-319592593 780370000
61498642 -341405295
273348736 -887131545
-140909764 -862666057
-141897045 -662091388
-83845248 434423350
-575419093 295531008
808415017 -768763308
448482118 -805421168
112071881 -321421927
937646372 -78509186
748722302 561120932
325854839 502509396
-142856052 533723122
937646372 -78509186
512017432 -606989172
630236821 -456599127
126923634 579875793
188081013 -895292396
-787355559 -407819218
-970130871 256310144
21133495 410002955
-232550855 -533238589
-324664136 671577332
-151497062 -52497185
728921043 64693367
429823428 808608130
460217304 -249223017
112391086 3860025
-151497062 -52497185
-486740171 522017653
-280640049 997452468
-175100225 291839964
930588836 20794645
