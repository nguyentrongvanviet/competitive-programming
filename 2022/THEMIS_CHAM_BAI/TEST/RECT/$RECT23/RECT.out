609304775882214360